app/views/users/show.html.haml

```diff
@@ -52,7 +52,7 @@
               %p.account-page-card-desc
                 = t('users.show.api_key_description', portal: portal_name)
               %div.apikey
-                = render ClipboardComponent.new(message: session[:user].apikey)
+                = render ClipboardComponent.new('icons/copy.svg', title: "Copy", message: session[:user].apikey)
```

Comment:
when you add the defaults to `ClipboardComponent` you will not need to change this